app/models/batch.rb

```diff
@@ -145,17 +145,25 @@ def ready_for_aspace_checks?
     resource_uri.present? && start_box.present? && end_box.present? && end_box >= start_box
   end
 
+  # Create an AbID for every top container. Manually increment the suffix after
+  # the first one because, as the transaction hasn't closed, each subsequent
+  # AbID can not query the database for what the most recent one is.
   def create_absolute_identifiers
     return unless absolute_identifiers.empty?
```

Comment:
There's still a check that this is empty, but the method doesn't put anything in it anymore. Not sure how this should work.

Reply:
I see, they're added to the batch below. So this just enforces that once a batch has absolute_identifiers it doesn't add more.